quests/quests.js

```diff
@@ -0,0 +1,62 @@
+import adventures from '../data/quest-data.js';
+// import { getUser, saveUser } from '../utils/storage-utils.js';
+import { findById } from '../utils/utils.js';
+import choiceCreator from './choices.js';
+import loadChar from '../char-state/char-state.js';
+import howGrumpy from './how-grumpy.js';
+// import howGrumpy from '../quests/how-grumpy';
+
+// get the character and stick it at the top
+loadChar();
+
+// WHY DO I HAVE THIS HERE grabs my section so i can append it
+const section = document.getElementById('section');
+// get adventures query param
+const searchParameters = new URLSearchParams(window.location.search);
+
+// get adventures quest ID
+const adventureID = searchParameters.get('id');
+// find the adventure
+const adventure = findById(adventures, adventureID);
+
+// make stuff
+// create a damn form
+
+// make h3 element
+const h3 = document.createElement('h3');
+// take adventure title and put in h3 textcontent
+h3.textcontent = adventure.title;
+// appendchild 
+section.append(h3);
```

Comment:
I feel like this should work. Since you're only appending one thing, it would semantically be appendChild...but this should still work.